server/bin/gold/test-0.1.txt

```diff
@@ -84,34 +71,30 @@ drwxrwxr-x          - public_html/results
 drwxrwxr-x          - public_html/static
 drwxrwxr-x          - public_html/static/css
 drwxrwxr-x          - public_html/static/css/v0.2
-drwxrwxr-x          - public_html/static/css/v0.2/css
--rw-rw-r--        308 public_html/static/css/v0.2/css/pbench_utils.css
+drwxr-xr-x          - public_html/static/css/v0.2/css
```

Comment:
Why has this changed? That should not be part of your changes at all.

Reply:
I think there must have been some problem. I corrected it